### PR TITLE
OSDOCS-10685_3#release note GCP

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Version: 
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-10685

Link to docs preview:
https://77621--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-installation-and-update-gcp-regions_release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by brendan-daly-red-hat** [comment]
@jianli-wei, Please review this release note.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -122,6 +122,10 @@ For more information, see the link:https://access.redhat.com/articles/7065136[Op
 
 With this release, you can enable bring your own public IPv4 addresses (BYOIP) feature when installing on AWS by using the `publicIpv4Pool` field to allocate Elastic IP addresses (EIPs). You must ensure that you have the xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[required permissions] to enable BYOIP. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
 
+[id="ocp-4-16-installation-and-update-gcp-regions_{context}"]
+==== Deploy GCP in the Dammam, Saudi Arabia and Johannesburg, South Africa regions
```
Comment (dfitzmau) [inline release_notes/ocp-4-16-release-notes.adoc:126]:
```suggestion
==== Deploy {gcp-short} in the Dammam (Saudi Arabia) and Johannesburg (South Africa) regions
```

Or

Deploy GCP in the Dammam, Saudi Arabia, and Johannesburg, South Africa, regions.

I find the commas more distracting here.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -122,6 +122,10 @@ For more information, see the link:https://access.redhat.com/articles/7065136[Op
...
+
+You can deploy {product-title} {product-version} in Google Cloud Platform (GCP) in the Dammam, Saudi Arabia, Middle East (`me-central2`) region and in the Johannesburg, South Africa (`africa-south1`) region. For more information, see xref:../installing/installing_gcp/installing-gcp-account.adoc#installation-gcp-regions_installing-gcp-account[Supported GCP regions].
```
Comment (dfitzmau) [inline release_notes/ocp-4-16-release-notes.adoc:128]:
```suggestion
You can deploy {product-title} {product-version} in {gcp-first} in the Dammam, Saudi Arabia, Middle East (`me-central2`) region and in the Johannesburg, South Africa (`africa-south1`) region. For more information, see xref:../installing/installing_gcp/installing-gcp-account.adoc#installation-gcp-regions_installing-gcp-account[Supported GCP regions].
```

Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:128]: Is the "Middle East" needed?

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**